Week01/Problem01/cbalusek_01.py

```diff
@@ -0,0 +1,18 @@
+#This project defines a function that takes any two numbers and sums their multiples to some cutoff value
+
+def sum(val1, val2, test):
```

Comment:
`sum()` is already a built-in Python function...check out the full list [here](https://docs.python.org/3/library/functions.html#built-in-functions).  So, it might be a good idea to call this function something else, like `sum_multiples()`, so it doesn't get confused for the built-in `sum()` function by anyone using your code.